erpnext_shopify/patches/V2_0/sync_missing_variants.py

```diff
@@ -0,0 +1,49 @@
+import frappe
+import requests.exceptions
+from erpnext_shopify.shopify_requests import post_request
+from frappe.utils import cint
+from erpnext_shopify.utils import disable_shopify_sync_for_item, make_shopify_log
+from erpnext_shopify.sync_products import get_erpnext_items_to_sync, get_variant_attributes
+
+def execute():
+	shopify_settings = frappe.get_doc("Shopify Settings")
+	
+	for item in get_erpnext_items_to_sync():
+		products = {}
+		variant_item_code_list = []
+				
+		if not cint(item.has_variants):
+			continue
+
+		variant_list, options, variant_item_name = get_variant_attributes(item, shopify_settings.price_list,
+			shopify_settings.warehouse, include_varaint_id=False)
+
+		variant_item_code_list.extend(variant_item_name)
+
+		erp_item = frappe.get_doc("Item", item.name)
+		
+		for idx, variant in enumerate(variant_list):
+			products["variant"] = variant
+			try:
+				new_variant = post_request("/admin/products/{0}/variants.json".format(item.get("shopify_product_id")),
+					products)
+
+				update_variant(new_variant, variant_item_code_list[idx])
+				frappe.db.commit()
+
+			except requests.exceptions.HTTPError, e:
```

Comment:
In disable_shopify_sync_for_item and make_shopify_log it will rollback the transactio